test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs

```diff
@@ -69,5 +64,33 @@ public ErrorPageTests(MvcTestFixture<ErrorPageMiddlewareWebSite.Startup> fixture
             Assert.Contains("/Views/ErrorFromViewImports/_ViewImports.cshtml", content);
             Assert.Contains(expectedMessage, content);
         }
+
+        [Fact]
+        public async Task RuntimeErrorAreListedByErrorPageMiddleware()
+        {
+            // The desktop CLR does not correctly read the stack trace from portable PDBs. However generating full pdbs
+            // is only supported on machines with CLSID_CorSymWriter available. On desktop, we'll skip this test on 
+            // machines without this component.
+#if NET451
+            if (!SymbolsUtility.SupportsFullPdbGeneration())
+            {
+                return;
+            }
+#endif
+
+            // Arrange
+            var expectedMessage = HtmlEncoder.Default.Encode("throw new Exception(\"Error from view\");");
```

Comment:
Suggest checking the surrounding HTML to make sure this line wasn't included because it was _near_ the one highlighted.